satellite/inspector/inspector_test.go

```diff
@@ -0,0 +1,109 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package inspector_test
+
+import (
+	"crypto/rand"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap/zaptest"
+
+	"storj.io/storj/internal/memory"
+	"storj.io/storj/internal/testcontext"
+	"storj.io/storj/internal/testplanet"
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/pkg/storj"
+	"storj.io/storj/satellite/inspector"
+	"storj.io/storj/storage"
+)
+
+func TestInspectorStats(t *testing.T) {
+	ctx := testcontext.New(t)
+	defer ctx.Cleanup()
+
+	planet, err := testplanet.New(t, 1, 6, 1)
+	require.NoError(t, err)
+	defer ctx.Check(planet.Shutdown)
+
+	planet.Start(ctx)
+
+	uplink := planet.Uplinks[0]
+	testData := make([]byte, 1*memory.MiB)
+	_, err = rand.Read(testData)
+	assert.NoError(t, err)
+
+	bucket := "testbucket"
+
+	err = uplink.Upload(ctx, planet.Satellites[0], bucket, "test/path", testData)
+	assert.NoError(t, err)
+
+	log := zaptest.NewLogger(t)
+
+	health, err := inspector.NewEndpoint(log, planet.Satellites[0].Overlay.Service, planet.Satellites[0].Metainfo.Service)
+	assert.NoError(t, err)
```

Comment:
Maybe we should stop one of the storagenodes such that we test actually seeing the stopped information.